subprojects/language-java/src/crossVersionTest/groovy/org/gradle/api/internal/tasks/compile/tooling/JavaCompileTaskOperationResultCrossVersionTest.groovy

```diff
@@ -87,6 +87,7 @@ class JavaCompileTaskOperationResultCrossVersionTest extends ToolingApiSpecifica
     @TargetGradleVersion(">=6.7")
     def "reports annotation processor results for JavaCompile task even when build event listener is used"() {
         settingsFile << """
+            import javax.inject.Inject
```

Comment:
:question: Why is this import required? Default imports should be available in settings scripts

Reply:
As far as I understand, we need the import because we try to run this test with 6.7 as well and without the implicit import support (6.8), this fails to compile. See[ build failure](https://builds.gradle.org/buildConfiguration/Gradle_Check_Quick_1_bucket19/38744565)

Not sure if we can do something about this

Reply:
Ah right. makes sense :+1: